runkit.js

```diff
@@ -1,22 +1,22 @@
-const levenbergMarquardt = require('ml-levenberg-marquardt');
+const levenbergMarquardt = require('ml-levenberg-marquardt').default;
```

Comment:
If you change the rollup config to default exports, you can remove this `.default`

Reply:
this is a great tip, thanks!